meta/terminal42/contao-url-rewrite/en.yml

```diff
@@ -0,0 +1,8 @@
+en:
+  title: Folder page
+  description: >
+    This extension provides a new way for Contao to set various URL rewrites via
+    config providers that take entries from a configuration file or from a backend module.
+  keywords:
+    - URL rewriting
+    - tool
```

Comment:
Maybe also `redirect` and because people love apache `htaccess`?

Reply:
Good idea. Update of PR will follow